terraform/trial_rg/services.tf

```diff
@@ -122,17 +128,32 @@ module "trial_app_service_init" {
     "WEBSITES_PORT"               = "8080" # The container is listening on 8080
   }
 
+
   depends_on = [
     azurerm_app_service_plan.apps_service_plan,
     module.trial_sc_config,
     module.trial_sc_discovery,
-    module.trial_app_service_site,
-    module.trial_app_service_practitioner,
-    module.trial_app_service_role,
     module.trial_sc_gateway,
   ]
 }
 
+data "azurerm_subscription" "current" {
```

Comment:
L140-142. are these needed?

Reply:
I think this is needed to allow other vars access to the subscription element?? As per documentation - 
data "azurerm_subscription" "current" {}

if this isnt defined how would other vars access the subscription element?